src/evergreen/api.py

```diff
@@ -1079,6 +1079,33 @@ def get_users_for_role(self, role: str) -> UsersForRole:
         url = self._create_url(f"/roles/{role}/users")
         return UsersForRole(self._call_api(url, method="GET").json(), self)
 
+    def all_user_permissions_for_resource(
+        self, resource_id: str, resource_type: PermissionableResourceType
+    ) -> Dict[str, Dict[str, int]]:
+        """
```

Comment:
It looks like [sphinx](https://www.sphinx-doc.org/en/master/) doesn't like the formatting of this code comment. You might want to try pulling the description of the return value before any of the tags (`:param` and `:return`). You also might want to try something like what is described [here](https://stackoverflow.com/questions/64451966/python-sphinx-how-to-embed-code-into-a-docstring) to ensure the json is formatted.